### DISCUSSION TITLE
Astronauts should cost $$ instead of $.

### DISCUSSION THREAD

**Discussion by sum1els-com** [jacobsalmela/ascii-planets#40] [Ideas]
Currently the rules allow direct conversion of $ into /.  This means that a Factory, which produces $, effectively produces either $ or /.  In the early game, it's much more efficient to build 4 Factories instead of 2 Factories and 2 Academies.  The benefit of being able to activate an Academy on a roll of 2 is not enough to overcome this.

Furthermore, Astronauts score 2 points at the end of the game and Currency only scores 1 point, so it is currently beneficial to convert as much Currency to Astronauts as possible as the game nears its end.  This seems like an unnecessary strategy to encourage.

**Comment by jacobsalmela** [comment]
Thank you again for the help!  

That all makes sense.  I think I simply held on to too much of the original game and Disney try enough new things--a consequence of trying to submit it to the contest on a deadline.

This is fun though--improving it into its own game.  I just needed a new set of eyes for it

**Comment by sum1els-com** [comment]
This is the part where I admit I haven't played ASCII Empires yet, since ASCII Planets sounded thematically more interesting to me.  I'm not sure whether this makes me less biased or just naive...

**Reply by jacobsalmela** [reply]
I suggest you do not play it until all of your feedback for this game has diminished!

**Comment by jacobsalmela** [comment]
This should be addressed in https://github.com/jacobsalmela/ascii-planets/releases/tag/v1.2.5. 

Feel free to mark this as an answer for anyone else landing here.
